www/src/data/sidebars/doc-links.yaml

```diff
@@ -174,8 +174,8 @@
                   link: /docs/creating-a-transformer-plugin/
                 - title: Submit to Plugin Library
                   link: /contributing/submit-to-plugin-library/
-                - title: Pixabay Source Plugin Tutorial
-                  link: /docs/pixabay-source-plugin-tutorial/
+                - title: Pixabay Image Source Plugin Tutorial
+                  link: /tutorial/pixabay-source-plugin-tutorial/
```

Comment:
@shannonbux this is worth a discussion around source plugin tutorials. Should they stay here, in the Creating Plugins section of the docs? Advanced Tutorials could then be more consumer-based and less creator-based.

Reply:
At one point we discussed splitting up "creator" (creating for Gatsby) oriented tutorials, and "consumer" (using Gatsby things) oriented tutorials, explicitly. @jlengstorf do you remember this?

Reply:
Yeah, I think it would be a good idea to move toward two contextual buckets: "Using Gatsby" and "Building for Gatsby".

That way we can have a bucket of "here's how to build plugins/themes/whatever" that is clearly an advanced, "how to customize Gatsby" area of the docs.

The goal is to keep the happy path free of noise, so for someone who just wants to build a site and add some plugins, they shouldn't ever need to see or care about how to _build_ plugins.

Reply:
I like that idea! I've moved the source plugin tutorial back to Docs > Plugins > Creating Plugins to limit the scope of this PR, and we can work on Building for Gatsby separately.